apps/workerexample/tasks.py

```diff
@@ -12,7 +12,7 @@ def say_hello(name):
     print('Hello', name)
 
 
-@task()
+#  @task()
```

Comment:
Is this supposed to be commented out?

Reply:
yea just while I was testing different variations of the test worker app